diff --git a/libavfilter/vf_fsppdsp.c b/libavfilter/vf_fsppdsp.c
index 8025e87366..5750ad8af1 100644
--- a/libavfilter/vf_fsppdsp.c
+++ b/libavfilter/vf_fsppdsp.c
@@ -52,7 +52,7 @@ enum {
     FIX_1_082392200   = FIX(1.082392200, 13),
 };
 
-DECLARE_ALIGNED(8, const uint8_t, ff_fspp_dither)[8][8] = {
+DECLARE_ASM_ALIGNED(8, const uint8_t, ff_fspp_dither)[8][8] = {
     {  0,  48,  12,  60,   3,  51,  15,  63, },
     { 32,  16,  44,  28,  35,  19,  47,  31, },
     {  8,  56,   4,  52,  11,  59,   7,  55, },
