--- pacutils/src/pacinfo.c
+++ pacutils/src/pacinfo.c
@@ -399,7 +399,6 @@
       prints("Packager:       %s\n", alpm_pkg_get_packager(pkg));
       printt("Build Date:     %s\n", alpm_pkg_get_builddate(pkg));
       printt("Install Date:   %s\n", alpm_pkg_get_installdate(pkg));
-      prints("MD5 Sum:        %s\n", alpm_pkg_get_md5sum(pkg));
       prints("SHA-256 Sum:    %s\n", alpm_pkg_get_sha256sum(pkg));
       if (verbosity >= 2) {
         prints("Signature:      %s\n", alpm_pkg_get_base64_sig(pkg));